static/include/services/apis/robot.md

```diff
@@ -2,17 +2,17 @@
 | Method Name                                                     | Description                                                                  |
 | --------------------------------------------------------------- | ---------------------------------------------------------------------------- |
 | [`Options.with_api_key`](/build/program/apis/robot/#optionswith_api_key) | Create Robot client connection options with an API key as credentials. |
-| [`AtAddress`](/build/program/apis/robot/#ataddress) | Create a robot client that is connected to the robot at the provided address. |
+| [`AtAddress`](/build/program/apis/robot/#ataddress) | Create a robot client that is connected to the machine at the provided address. |
 | [`WithChannel`](/build/program/apis/robot/#withchannel) | Create a robot that is connected to a robot over the given channel. |
```

Comment:
(opt) Also could update phrasing here

Reply:
```suggestion
| [`WithChannel`](/build/program/apis/robot/#withchannel) | Create a machine that is connected to a machine over the given channel. |
```
Okay yeah--but while we're at it either way "robot connected to robot" or "machine connected to machine", this is actually kinda confusing and sounds circular--do you happen to know if there's a better way to phrase this overall? Not sure what it actually does

Reply:
```suggestion
| [`WithChannel`](/build/program/apis/robot/#withchannel) | Create a RobotClient that is connected to a machine over the given channel. |
```
JK will use phrasing from the instance above. TY!